### ISSUE TITLE
CSS problems after the version 6.7.0

### ISSUE THREAD

**Issue by isatanis** [onfido/onfido-sdk-ui#1391]
Hey guys, we're using this SDK on our React project. We have no problem until your latest build. It breaks our css. You can see the related css below.

````
a, .onfido-sdk-ui-Theme-link, .onfido-sdk-ui-Theme-textButton {
    font-family: var(--osdk-font-family-body);
    text-decoration: none;
    border: 1px solid transparent;
    /* border-bottom-color: var(--osdk-color-border-link-underline); */
    background-color: transparent;
    color: var(--osdk-color-content-body);
}

a:not(:disabled):hover, .onfido-sdk-ui-Theme-link:not(:disabled):hover, .onfido-sdk-ui-Theme-textButton:not(:disabled):hover {
    background-color: var(--osdk-color-background-link-hover);
    color: var(--osdk-color-content-link-text-hover);
}
````

Since the "a" tag's CSS is used directly in this way, it affects globally.

I've seen these two so far, I don't know if anything else like these. 

For now, we're using version 6.6.0.

### Version info:

- Onfido SDK: 6.7.0
- OS: macOS 11.0.1
- Browser: Chrome
- Node: 10.14.2
- Npm: 7.7.5

An example:
![Mar-26-2021 12-45-45](https://user-images.githubusercontent.com/43003459/112613578-5117fb80-8e31-11eb-87e2-e73f221f339f.gif)

**Comment by stefaniacardenas** [comment]
Hi @isatanis, thanks for letting us know, we'll do a patch release to fix this as soon as possible.

**Comment by stefaniacardenas** [comment]
Hi @isatanis, we've released a fix in version `6.7.1`. Let me know if everything is working as expected on upgrade. Again, thank you so much for flagging the issue.
